helm_deploy/dps-smoketest/templates/_envs.tpl

```diff
@@ -43,4 +43,8 @@ env:
         name: {{ template "app.name" . }}
         key: APPINSIGHTS_INSTRUMENTATIONKEY
 
+  - name: APPLICATIONINSIGHTS_CONNECTION_STRING
```

Comment:
the name above needs changing from `APPLICATION_INSIGHTS_IKEY` to `APPINSIGHTS_INSTRUMENTATIONKEY`

Reply:
Done